### PR TITLE
VEN-967 | Show success notification when assigning berth

### PR DESCRIPTION
**Pull request by ahie**
## Description :sparkles:

Adds a success toast when assigning berths to customers.

### Closes :no_good_woman:

[VEN-967](https://helsinkisolutionoffice.atlassian.net/browse/VEN-967)

### Manual testing :construction_worker_man:

- Open applications list
- Select an unhandled application
- Go through the application handling flow, assign a suitable berth
- After assigning, the user should be redirected back to the applications list and a success toast with berth and customer information should be shown

### CODE REVIEW

File: src/features/offer/OfferContainer.tsx [hunk 1]
```diff
@@ -64,15 +66,28 @@ const OfferContainer = () => {
...
-          berthId,
+          berthId: berth.id,
         },
       },
-    }).then(() => history.push('/applications'));
+    }).then(() => {
+      history.push('/applications');
+      hdsToast({
+        type: 'success',
+        autoDismiss: false,
```
Comment (santtul) [inline src/features/offer/OfferContainer.tsx:81]: Should we autoDismiss it?

**Review by santtul — CHANGES_REQUESTED** [review]
One question there.

File: src/features/offer/OfferContainer.tsx (hunk 1 above)
Comment (ahie) [reply src/features/offer/OfferContainer.tsx:81]: The autodismiss is so quick I didn't have time to verify from the text that it was actually assigned correctly.

Comment (santtul) [reply src/features/offer/OfferContainer.tsx:81]: Fair enough.

**Review by santtul — APPROVED** [review]
👍

File: src/features/offer/OfferContainer.tsx (hunk 1 above)
Comment (ahie) [reply src/features/offer/OfferContainer.tsx:81]: Added a way to configure the dismiss time and increased it a bit for this toast.

**Pull request merged**